pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -101,6 +101,32 @@ def test_to_datetime_format_YYYYMMDD(self, cache):
         expected = Series(["20121231", "20141231", "NaT"], dtype="M8[ns]")
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "input_s",
+        [
+            # None with Strings
+            Series(["19801222", None, "20010112", np.nan, "NaT", pd.NaT]),
```

Comment:
not what mean, use *another* layer of paramterization; you should have 8 cases total (2 series x 4 of [None, np.nan, 'NaT', np.NaT])

Reply:
Now that we have 8 test cases